.devcontainer/README.md

```diff
@@ -0,0 +1,27 @@
+# Dev Containers
+
+## Image Used in This Project
+
+This project uses a Nextcloud Docker image for the development container. The image is defined in the `.devcontainer.json` configuration file. For more details about the image and its configuration, you can refer to the [GitHub repository](https://github.com/juliusknorr/nextcloud-dev).
+
+# VS Code
+
+## Introduction to Dev Containers
+
+Dev Containers are a feature in Visual Studio Code that allows you to develop inside a containerized environment. This ensures that your development environment is consistent across different machines and setups. By using Dev Containers, you can easily share your development environment with your team, ensuring that everyone is working with the same dependencies and configurations.
+
+## How to Use Dev Containers with VS Code
```

Comment:
I think we should leave this to the upstream documentation. The F1 for example is not my button to open the command palette.